langchain4j/src/test/java/dev/langchain4j/service/AiServicesUserMessageConfigTest.java

```diff
@@ -47,6 +53,18 @@ interface AiService {
         @UserMessage("What is the {{it}} of {{country}}?")
         String chat7(@V("it") String it, @V("country") String country);
 
+        @UserMessage("Count the number of cars in this image")
+        String chat8(@UserMessage ImageContent imageContent);
```

Comment:
Perhaps consider returning `int` here since the expected output should always be an int.